lib/jazz_fingers/prompt.rb

```diff
@@ -32,44 +34,55 @@ def separator
       red_text(@separator)
     end
 
-    def name
-      blue_text(@application_name)
+    def context(module_name = "main")
+      name =
+        if module_name == "main"
+          @application_name
+        else
+          abbreviated_context(module_name)
+        end
+
+      blue_text("(#{name})")
     end
 
     def line_number(pry)
       if pry.respond_to? :input_ring
-        "[#{bold_text(pry.input_ring.size)}]"
+        bold_text(pry.input_ring.size)
       else
-        "[#{bold_text(pry.input_array.size)}]"
+        bold_text(pry.input_array.size)
       end
     end
 
-    def text(object, level)
-      level = 0 if level < 0
```

Comment:
This method (`#text`) doesn't appear to be used anywhere, so I removed it — let me know if I should revert this change.